### PR TITLE
DAOS-7013 obj: OIT should match with cont_rf

### PR DESCRIPTION
**Pull request by liuxuezhao**
OIT should with #replicas = cont_rf + 1.

Signed-off-by: Xuezhao Liu <xuezhao.liu@intel.com>

### CODE REVIEW

File: src/include/daos/object.h [hunk 1]
```diff
@@ -103,12 +103,13 @@ enum {
 	 *
 	 * NB: it should be smaller than OC_BACK_COMPAT (50)
 	 */
-	DAOS_OC_OIT_V1	= 45,
+	DAOS_OC_OIT_RF0	= 45,
+	DAOS_OC_OIT_RF1	= 46,
+	DAOS_OC_OIT_RF2	= 47,
+	DAOS_OC_OIT_RF3	= 48,
+	DAOS_OC_OIT_RF4	= 49,
```
Comment (Nasf-Fan) [inline src/include/daos/object.h:110]: Why need to define so many OIT class? does it waste the reserved object class?

**Review by Nasf-Fan — APPROVED** [review]

File: src/include/daos/object.h (hunk 1 above)
Comment (liuxuezhao) [reply src/include/daos/object.h:110]: for cont_rf we only with 5 - RF0 - RF4, so using 5 OC (adding 4) looks fine?

**Comment by liuxuezhao** [comment]
test failure "unable to contact the DAOS Management Service" due to https://jira.hpdd.intel.com/browse/DAOS-7016

File: src/include/daos/object.h (hunk 1 above)
Comment (Nasf-Fan) [reply src/include/daos/object.h:110]: It's fine for me, just not sure whether we will use some of them or not, such as DAOS_OC_OIT_RF4.

**Review by gnailzenh — APPROVED** [review]

**Pull request merged**